src/api/app/views/message/_messages.xml.builder

```diff
@@ -9,8 +9,8 @@ xml.messages(root_attrs) do
     attrs[:severity] = msg.severity || 0
     attrs[:sent_at] = msg.sent_at.xmlschema if msg.sent_at
     attrs[:user] =  msg.user.login if msg.user
-    msg.send_mail ? attrs[:send_mail] = true : attrs[:send_mail] = false
-    msg.private ? attrs[:private] = true : attrs[:private] = false
+    attrs[:send_mail] = msg.send_mail ? true : false
```

Comment:
`send_mail` and `private` are [boolean fields](https://github.com/openSUSE/open-build-service/blob/327a0b2cb5670d6a0b105a91f6b46bcd36ebf7a2/src/api/app/models/message.rb#L13-L14).

```suggestion
    attrs[:send_mail] = msg.send_mail
```